ui/src/state/chat/chat.ts

```diff
@@ -986,6 +986,17 @@ export function useMultiDms() {
   return dms;
 }
 
+const selDms = (s: ChatState) => s.dms;
+export function useDms() {
+  const dms = useChatState(selDms);
+
+  useEffect(() => {
+    useChatState.getState().fetchDms();
```

Comment:
we should have already fetched DMs on load or seen any come in on subscriptions. I think the `useMultiDms` above was from a time when we didn't do that on load for multi dms, hence the fetch, but we do now so it just needs to be corrected.

Reply:
It won't load if we're in groups though, correct? So we'd still need a fetch in both cases if we're in groups.

Reply:
pushing up a PR where we check if DMs are already in state before returning.